src/common/plugins/plugins-config-store.js

```diff
@@ -88,36 +87,43 @@ const removeUndefineds = (obj: Object = {}): Object =>
     return product;
   }, {});
 
-const pluginConfig = {
+class pluginConfigStore {
```

Comment:
If it's a class now, change to CamelCase

Reply:
done